site/config.toml

```diff
@@ -47,12 +47,8 @@ pygmentsStyle = "tango"
 [permalinks]
 blog = "/:section/:year/:month/:day/:slug/"
 
-## Configuration for BlackFriday markdown parser: https://github.com/russross/blackfriday
-[blackfriday]
-plainIDAnchors = true
-hrefTargetBlank = false
-angledQuotes = false
-latexDashes = true
+[markup.goldmark.renderer]
+unsafe= true
```

Comment:
nit: space before = sign?

Reply:
oh yeah, that's ugly. I'll fix that up. Copy paste from stack overflow issue!